drivers/tty/serdev/pibridge_trace.h

```diff
@@ -32,7 +42,7 @@ DECLARE_EVENT_CLASS(pibridge_gate_header_class,
 		__entry->len = hdr->len;
 	),
 	TP_printk(
-		"dst=%u src=%u cmd=%u seq=%u len=%u",
+		"dst=%u src=%u cmd=0x%02x seq=%u len=%u",
```

Comment:
Should probably be %04x because the command is a 16-bit value. (Granted, only 5 bits are used by the currently defined commands so far...)